### DISCUSSION TITLE
Gifs playing in local build, but not in production

### DISCUSSION THREAD

**Discussion by osebd-aqua** [vercel/next.js#45698] [Help]
Hello community,

We currently have a problem where loading gifs from an external source (via url) in our nextJs app does not render the gifs properly in production.

Any help would be much appreciated.

Below is the snippet with image configuration taken from next.config.js file.

  images: {
    unoptimized: true,
    formats: ["image/avif", "image/webp"],
    dangerouslyAllowSVG: true,
    contentSecurityPolicy: "default-src 'self'; script-src 'none'; sandbox;",
    domains: [
      "storage.googleapis.com",
    ],
  },

**Comment by ciruz** [comment]
Somebody had a similar issue with gifs on Azure, but it was a azure configuration issue.

https://github.com/vercel/next.js/issues/34310

It's probably not the same issue but maybe it helps you.

**Comment by guitheengineer** [comment]
Can you share what error you get? It can be a lot of things, including if the gifs are too big
